pkgs/by-name/sp/spice-autorandr/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoconf
+, automake
+, libX11
+, libXrandr
+}:
+
+stdenv.mkDerivation rec {
+  name = "spice-autorandr";
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "seife";
+    repo = name;
+    rev = "0f61dc9";
+    hash = "sha256-eBvzalWT3xI8+uNns0/ZyRes91ePpj0beKb8UBVqo0E=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ autoconf automake libX11 libXrandr ];
```

Comment:
I think `autoconf` and `automake` are already per default in the buildInputs. Can you try without?

Reply:
Now using `autoreconfHook` instead of listing `autoconf` and `automake` explicitly, as per https://github.com/NixOS/nixpkgs/pull/259784#discussion_r1349780159